pandas/tests/scalar/timestamp/test_constructors.py

```diff
@@ -353,25 +367,28 @@ def test_out_of_bounds_value(self):
         Timestamp(min_ts_us)
         Timestamp(max_ts_us)
 
+        msg = "Out of bounds"
         # One us less than the minimum is an error
-        with pytest.raises(ValueError):
+        with pytest.raises(ValueError, match=msg):
             Timestamp(min_ts_us - one_us)
 
         # One us more than the maximum is an error
-        with pytest.raises(ValueError):
+        with pytest.raises(ValueError, match=msg):
             Timestamp(max_ts_us + one_us)
 
     def test_out_of_bounds_string(self):
-        with pytest.raises(ValueError):
+        msg = "Out of bounds"
+        with pytest.raises(ValueError, match=msg):
             Timestamp("1676-01-01")
-        with pytest.raises(ValueError):
+        with pytest.raises(ValueError, match=msg):
             Timestamp("2263-01-01")
 
     def test_barely_out_of_bounds(self):
         # GH#19529
         # GH#19382 close enough to bounds that dropping nanos would result
         # in an in-bounds datetime
-        with pytest.raises(OutOfBoundsDatetime):
+        msg = "Out of bounds"
```

Comment:
is there more to this message?

Reply:
the full message is "Out of bounds nanosecond timestamp: 2262-04-11 23:47:16".
I will add the full message if needed to. Looking at the old code from other places I thought this is enough.